### DISCUSSION TITLE
What are "preambles"?

### DISCUSSION THREAD

**Discussion by MichalSznajder** [linq2db/linq2db#3609] [Q&A] [answered]
I found a [bug ](https://github.com/linq2db/linq2db.EntityFrameworkCore/issues/237) in sister project of `linq2db.EntitiyFrameworkCore`. It manifests in deadlock of an application in some scenarios. I wanted to fix it so I was analyzing code of `linq2db.EFC`  and also had a look in `linq2db`. I wonder what are "preambles" in `linq2db`?

**Comment by MaceWindu** [comment, marked as answer]
Preambules are additional queries for eager load "query"
